### PR TITLE
Plugins: Add feat toggle to install managed plugins

### PR DESCRIPTION
**Pull request by oshirohugo**
**What is this feature?**

Add feat toggle to install managed plugins

**Why do we need this feature?**

This is necessary to control the availability of new feature https://github.com/grafana/grafana-plugins-platform-team/issues/21
release.

**Who is this feature for?**

Grafana Developers

**Which issue(s) does this PR fix?**:

This is part of https://github.com/grafana/grafana-plugins-platform-team/issues/21

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: pkg/services/featuremgmt/registry.go [hunk 1]
```diff
@@ -904,5 +904,12 @@ var (
 			FrontendOnly: false,
 			Owner:        grafanaDatabasesFrontend,
 		},
+		{
+			Name:            "managedPluginsInstall",
+			Description:     "Install managed plugins directly from plugins catalog",
+			Stage:           FeatureStageExperimental,
+			RequiresDevMode: true,
```
Comment (marefr) [inline pkg/services/featuremgmt/registry.go:911]: Nit. Is this true?

**Review by marefr — APPROVED** [review]
LGTM with a comment/question

File: pkg/services/featuremgmt/registry.go (hunk 1 above)
Comment (oshirohugo) [reply pkg/services/featuremgmt/registry.go:911]: I wanted to be safe and test it before going to production.
Will this make my installation tests more difficult to perform? (I don't know much about our environments)

Comment (oshirohugo) [reply pkg/services/featuremgmt/registry.go]: Changed anyway XD

**Review by andresmgot — APPROVED** [review]

File: pkg/services/featuremgmt/registry.go (hunk 1 above)
Comment (marefr) [reply pkg/services/featuremgmt/registry.go]: Would require app_mode=development I think which is not enabled in cloud. Merging this the feature toggle will be disabled unless configured to be enabled

Comment (oshirohugo) [reply pkg/services/featuremgmt/registry.go]: Tks for pointing this out!

**Pull request merged**